shimmingtoolbox/load_nifti.py

```diff
@@ -243,22 +260,22 @@ def scale_tfl_b1(image, json_data):
     # Reorder data shuffled by dm2niix into shape (x, y , n_slices*n_channels)
     b1_mag_vector = np.zeros((image.shape[0], image.shape[1], n_slices * n_channels))
     b1_phase_vector = np.zeros((image.shape[0], image.shape[1], n_slices * n_channels))
-    if 'ImageOrientationPatientDICOM' in json_data:
-        orientation = json_data['ImageOrientationPatientDICOM']
-        # Axial or Coronal cases
-        if orientation[:3] == [1, 0, 0]:
+    if 'ImageOrientationText' in json_data:
```

Comment:
I added a function to find the orientation of an acquisition, since the `ImageOrientationText` field comes from a private dice header field that is not always populated, I suggest having a fallback method that uses the function get_main_orientation() that was implemented in #323

Reply:
@po09i I updated everything as we discussed earlier